subsys/net/lib/nrf_cloud/coap/cddl/nrf_cloud_coap_agps.cddl

```diff
@@ -8,16 +8,14 @@
 ; AGPS Request
 ; ************
 
-type = rtAssistance/custom
-
 agps_req = {
-	? customTypes => [1*10 int],
+	? customTypes => tstr,
```

Comment:
If we do not do your requested change, what does your proxy's automatic conversion output for the value of `customTypes`?

Note that this ticket indicates that Cole changed the AGPS REST API so that it will not only accept query parameters but also a JSON payload.  Does this help your side out?  It accepts the customTypes as a JSON array of numbers: `"types":[1,2,8]".

Reply:
Yes, that would help. BTW, what is the ticket that will implement it?
BTW, I just noticed that `customTypes` parameter is deprecated in favour of `types`, we should update that as well.

Reply:
Sorry, forgot the link :-(
https://projecttools.nordicsemi.no/jira/browse/IRIS-5888

Reply:
I misunderstood IRIS-5888.  There is no plan for JSON unless Cole changes his mind.

Reply:
I will discuss this with them tomorrow (Wednesday).

Reply:
OK, this request is more viable.  It would be a little different in that they already plan to add a new REST endpoint for GNSS to handle multiple satellite constellations, not just GPS.  In this new endpoint, they are willing to accept JSON in the body of a POST rather than as a query parameter.

The timeline for this is sometime before July.

We will need to make due with glue logic conversion in your proxy from CBOR to REST query parameters, but can remove that once the new GNSS endpoint is ready.